charts/argocd-notifications/values.yaml

```diff
@@ -155,3 +158,41 @@ triggers:
   #   template: my-custom-template
   #
   # For more information: https://argoproj-labs.github.io/argocd-notifications/triggers_and_templates/
+
+bots:
+  # For more information: https://argoproj-labs.github.io/argocd-notifications/recipients/bot/
+  slack:
+    updateStrategy:
```

Comment:
There should be a boolean value whether the Slack bot is enabled that controls whether the `Deployment` and the other resources needed for the bot are provisioned.